src/pharmpy/modeling/plots.py

```diff
@@ -639,9 +641,9 @@ def _vertical_line():
     return line
 
 
-def _identity_line(xmin, xmax, ymin, ymax):
-    x1, x2 = alt.param(value=xmin), alt.param(value=xmax)
-    y1, y2 = alt.param(value=ymin), alt.param(value=ymax)
+def _identity_line(min_value, max_value):
+    x1, x2 = alt.param(value=min_value), alt.param(value=max_value)
+    y1, y2 = alt.param(value=min_value), alt.param(value=max_value)
```

Comment:
Ok, so it was changed in another commit.  (Consider using `git --amend` to update the first commit in the future)